react-components/src/tools/styles.js

```diff
@@ -10,4 +12,32 @@ export default {
     addBtn: {
         marginRight: "10px",
     },
-};
+
+    toolbar: {
+        backgroundColor: palette.lightGray,
+        "& div": {
+            marginRight: "10px",
```

Comment:
I'll switch to that. Also, FYI, this is happening:
```
Warning: Material-UI: theme.spacing.unit usage has been deprecated.
It will be removed in v5.
You can replace `theme.spacing.unit * y` with `theme.spacing(y)`.
```